packages/editor/src/utils/index.js

```diff
@@ -6,3 +6,4 @@ import mediaUpload from './media-upload';
 export { mediaUpload };
 export { cleanForSlug } from './url.js';
 export { getTemplatePartIcon } from './get-template-part-icon';
```

Comment:
It's funny that we already have a `getTemplatePartIcon`. It must be something that is very similar no?

Reply:
They are the same 😁, but I don't think that we can import `getTemplatePartIcon` from `editor` package in `block-library` package.